### DISCUSSION TITLE
Public Feed URL is there a way to customize the output on the fly

### DISCUSSION THREAD

**Discussion by Wenzelranch** [code4recovery/12-step-meeting-list#801] [Q&A] [answered]
I am asking if there is a way to customize the feed url to get it to show a certain subset of meetings.
example I want to offer a feed link to another site but would need it filtered by City or District or Location,
is that possible at all? I have tried searching for an answer but have come up empty.
our feed 
https://racinecentraloffice.org/wp-admin/admin-ajax.php?action=meetings
are there any other things or switches that could be appended to the end of this that would give me the results I am hoping for?
Example, pull all meetings from the feed for districtX.

**Comment by joshreisner** [comment, marked as answer]
hi @Wenzelranch yes this should be possible actually!

https://racinecentraloffice.org/wp-admin/admin-ajax.php?action=meetings&district=foo

where "foo" is the "slug" of the district, which you can find on the Districts page
